eclipse-che/che-dashboard
# Add factory create policies for devWorkspaces

Signed-off-by: Oleksii Orel <oorel@redhat.com>



### What does this PR do?
Add factory create policies for devWorkspaces.

### What issues does this PR fix or reference?
fixes https://github.com/eclipse/che/issues/19711




### Steps for testing this PR: 
1. Create several workspaces from the same sample. 

* expected result: Workspace names should have generated parts at the end.

 2. Resolve the factory with 'policies.create=peruser'
```{che-server}/f?url=https://raw.githubusercontent.com/che-samples/java-spring-petclinic/devfilev2/devfile.yaml&policies.create=peruser```
several times.

* expected result: A new workspace was created only in the first attempt.
